server/test/framework/Syntax_Tests.cpp

```diff
@@ -3225,6 +3225,44 @@ namespace {
         );
     }
 
+    /*TEST_F(Syntax_Test, run_tests_for_input_output_c) {
```

Comment:
Why you add commented test? If you need it in future, but don't want to run now use `DISABLED_` prefix for test name.